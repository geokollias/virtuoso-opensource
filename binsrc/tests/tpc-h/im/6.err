Starting Impala Shell without Kerberos authentication
Connected to ip-10-0-1-251.ec2.internal:21000
Server version: impalad version 2.1.3-cdh5 RELEASE (build 20816e26a150d20d2c92c470aa40b342521b243e)
Query: use TPCH
Query: select sum(l_extendedprice * l_discount) as revenue
from
	lineitem
where
	l_shipdate >= '1994-01-01'
	and l_shipdate < cast ('1994-01-01' as timestamp) + INTERVAL 1 YEAR
	and l_discount between .06 - 0.01 and .06 + 0.010001
	and l_quantity < 24
Fetched 1 row(s) in 13.36s
