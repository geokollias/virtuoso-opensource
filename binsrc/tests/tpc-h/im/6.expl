[?1034h+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Explain String                                                                                                                                                                       |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Estimated Per-Host Requirements: Memory=362.00MB VCores=1                                                                                                                            |
|                                                                                                                                                                                      |
| 03:AGGREGATE [FINALIZE]                                                                                                                                                              |
| |  output: sum:merge(l_extendedprice * l_discount)                                                                                                                                   |
| |  hosts=1 per-host-mem=unavailable                                                                                                                                                  |
| |  tuple-ids=1 row-size=8B cardinality=1                                                                                                                                             |
| |                                                                                                                                                                                    |
| 02:EXCHANGE [UNPARTITIONED]                                                                                                                                                          |
| |  hosts=1 per-host-mem=unavailable                                                                                                                                                  |
| |  tuple-ids=1 row-size=8B cardinality=1                                                                                                                                             |
| |                                                                                                                                                                                    |
| 01:AGGREGATE                                                                                                                                                                         |
| |  output: sum(l_extendedprice * l_discount)                                                                                                                                         |
| |  hosts=1 per-host-mem=10.00MB                                                                                                                                                      |
| |  tuple-ids=1 row-size=8B cardinality=1                                                                                                                                             |
| |                                                                                                                                                                                    |
| 00:SCAN HDFS [tpch.lineitem, RANDOM]                                                                                                                                                 |
|    partitions=1/1 files=82 size=20.20GB                                                                                                                                              |
|    predicates: l_shipdate >= '1994-01-01', l_shipdate < CAST('1994-01-01' AS TIMESTAMP) + INTERVAL 1 YEAR, l_discount >= 0.06 - 0.01, l_discount <= 0.06 + 0.010001, l_quantity < 24 |
|    table stats: 600037902 rows total                                                                                                                                                 |
|    column stats: all                                                                                                                                                                 |
|    hosts=1 per-host-mem=352.00MB                                                                                                                                                     |
|    tuple-ids=0 row-size=40B cardinality=6000                                                                                                                                         |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
