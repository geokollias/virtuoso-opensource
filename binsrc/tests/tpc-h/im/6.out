[?1034h+------------------+
| revenue          |
+------------------+
| 12330426888.4643 |
+------------------+
